### DISCUSSION TITLE
Works with  AzureAD

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#376] [Feature Requests & Suggestions]
I want a mechanism that can work with AzureAD. I imagine that there are many users who use Azure OpenAI, so AD linkage is important

Is it possible to achieve this easily using the following?
https://github.com/ahaenggli/AzureAD-LDAP-wrapper

**Comment by danny-avila** [comment]
This is really out of my knowledge, as I've only managed to getting Azure working with limited access to someone's org.  I am completely unfamiliar to what you linked to.

**Comment by tttll1091** [comment]
thank you chatgpt clone successfully booted using azure virtual machine 
What kind of setting is the access control by azuread?

**Comment by yishaik** [comment]
Feels like you'll have to do something like that
https://learn.microsoft.com/en-us/azure/app-service/configure-authentication-provider-aad?tabs=workforce-tenant
You'll need to have an Enterprise app in AAD and manage authorization over there

**Comment by danny-avila** [comment]
@tttll1091 if you’ve got something working that you think others can benefit from, a write up in our docs directory would be very much appreciated!
